teerex/sgx-verify/src/lib.rs

```diff
@@ -478,9 +479,27 @@ pub fn deserialize_tcb_info(
 	signature: &[u8],
 	certificate: &webpki::EndEntityCert,
 ) -> Result<TcbInfo, &'static str> {
+	log::info!(
+		target: TEEREX,
+		"Called into runtime call register_tcb_info(), inside Self::deserialize_tcb_info."
```

Comment:
I am not sure if `Called into runtime call ...` is right because this assumes some context, which is not always true. It should be evident from the sequence in the logs, that we entered from the `register_tcb_info`.

Reply:
Yep, removed them!